build/web/ci-common.sh

```diff
@@ -8,7 +8,7 @@ export PATH="$PWD:$PATH"
 # The Kokoro machines have Python 3.6.3 installed. Let's verify that here, and install the
 # distributions required for web/docs.
 python3 --version
-pip3 install mistletoe pygments jsbeautifier
+pip3 install mistletoe pygments jsbeautifier future_fstrings
```

Comment:
Some corporate Linux machines have Python 3.5 instead of 3.6 so I had to add this fstrings thing.